README.md

```diff
@@ -0,0 +1,49 @@
+<div align="center">
+<!-- Title: -->
+  <a href="https://github.com/TheAlgorithms/">
+    <img src="https://raw.githubusercontent.com/TheAlgorithms/website/1cd824df116b27029f17c2d1b42d81731f28a920/public/logo.svg" height="100">
+  </a>
+  <h1><a href="https://github.com/TheAlgorithms/">The Algorithms</a> - Nim</h1>
+<!-- Labels: -->
+  <!-- First row: -->
+  <a href="https://gitpod.io/#https://github.com/TheAlgorithms/Nim">
+    <img src="https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod&style=flat-square" height="20" alt="Gitpod Ready-to-Code">
+  </a>
+  <a href="https://github.com/TheAlgorithms/Nim/blob/master/CONTRIBUTING.md">
+    <img src="https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=0059b3&style=flat-square" height="20" alt="Contributions Welcome">
+  </a>
+  <img src="https://img.shields.io/github/repo-size/TheAlgorithms/Nim.svg?label=Repo%20size&style=flat-square" height="20">
+  <a href="https://discord.gg/c7MnfGFGa6">
+    <img src="https://img.shields.io/discord/808045925556682782.svg?logo=discord&colorB=7289DA&style=flat-square" height="20" alt="Discord chat">
+  </a>
+  <a href="https://gitter.im/TheAlgorithms">
+    <img src="https://img.shields.io/badge/Chat-Gitter-ff69b4.svg?label=Chat&logo=gitter&style=flat-square" height="20" alt="Gitter chat">
+  </a>
+  <!-- Second row: -->
+  <br>
+  <a href="https://github.com/TheAlgorithms/Nim/actions">
+    <img src="https://img.shields.io/github/actions/workflow/status/TheAlgorithms/Nim/build.yml?branch=master&label=CI&logo=github&style=flat-square" height="20" alt="GitHub Workflow Status">
+  </a>
+  <a href="https://github.com/pre-commit/pre-commit">
+    <img src="https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white&style=flat-square" height="20" alt="pre-commit">
+  </a>
+  <a href="https://github.com/psf/black">
+    <img src="https://img.shields.io/static/v1?label=code%20style&message=black&color=black&style=flat-square" height="20" alt="code style: black">
+  </a>
```

Comment:
These can be removed as we don't have CI yet, nor do we use pre-commit or the Python formatter.
```suggestion
```